lib/utils/locale-strings.js

```diff
@@ -11,7 +11,7 @@ const CALENDAR_STRINGS = {};
  */
 export function getCalendarStringsForLocale(locale) {
   if (!(/^[a-z]{2,3}(-[A-Za-z]{2})?$/).test(locale)) {
-    locale = navigator.language;
+    locale = navigator ? navigator.language : 'en';
```

Comment:
if i didn't miss anything, we should've check `typeof navigator`. Just checking `navigator` would also raise an exception 🤔